.editorconfig

```diff
@@ -3,9 +3,12 @@
 root = true
```

Comment:
All of this can be achieved by combination of eslint and prettier. The thing is that if you open and start coding in vim (with EditorConfig and without eslint + prettier) for instance you won't be notified of formatting errors.